sorc/build_ncep_post.sh

```diff
@@ -8,44 +7,103 @@ SHELL=/bin/sh
 # 01/16 Lin Gan:	Update to use GFS Vertical Structure
 # 07/16 J. Carley:      Generalize for other machines using modules
 # 07/18 Wen Meng:       Set post to v8.0.0 for fv3gfs
+# 10/19 M Kavulich:     Provide machine name as an input argument
 #
 #####################################################################################################
 #####################################################################################################
 
+#List of valid machines:
+validmachines=(theia jet wcoss_dell_p3 wcoss cray-intel hera)
```

Comment:
@fossell  You may also add orion and odin in the list.

Reply:
@WenMeng-NOAA - Done!

Reply:
@WenMeng-NOAA - Done!